articles/iot-edge/how-to-store-data-blob.md

```diff
@@ -198,6 +198,8 @@ You can use [Azure Storage Explorer](https://azure.microsoft.com/features/storag
 1. Download and install Azure Storage Explorer
 
 1. Connect to Azure Storage using a connection string
+   > [!NOTE]
+   > The latest version of Azure Storage Explorer uses a newer storage API version by default than the blob storage module supports. After Storage Explorer starts, select the Edit menu, and check to see if Target Azure Stack Hub APIs is selected. If it isn't, select Target Azure Stack Hub, and then restart Storage Explorer for the change to take effect. This configuration is required for compatibility with your IoT Edge environment.
```

Comment:
Hi @jessevl,

Thanks for the update. 

Just curious why you added the note to the *Connect using connection string* step. Would it be better to add a *Start* step or add the note to the previous step?

Suggested rewording. Remove note. Add new step after *Download and install* step.

```suggestion
1. The latest version of Azure Storage Explorer uses a newer storage API version not supported by the blob storage module. Start Azure Storage Explorer. Select the **Edit** menu. Verify the **Target Azure Stack Hub APIs** is selected. If it isn't, select **Target Azure Stack Hub**. Restart Azure Storage Explorer for the change to take effect. This configuration is required for compatibility with your IoT Edge environment.
```